system-tables/system-table-information-schema.md

```diff
@@ -470,13 +493,44 @@ desc information_schema.cluster_slow_query;
 | Cop_wait_addr             | varchar(64)         | YES  |     | <null>  |       |
 | Mem_max                   | bigint(20) unsigned | YES  |     | <null>  |       |
 | Succ                      | tinyint(1) unsigned | YES  |     | <null>  |       |
+| Plan_from_cache           | tinyint(1)          | YES  |     | <null>  |       |
 | Plan                      | longblob unsigned   | YES  |     | <null>  |       |
 | Plan_digest               | varchar(128)        | YES  |     | <null>  |       |
 | Prev_stmt                 | longblob unsigned   | YES  |     | <null>  |       |
 | Query                     | longblob unsigned   | YES  |     | <null>  |       |
 +---------------------------+---------------------+------+-----+---------+-------+
 ```
 
+When the cluster system table is queried, TiDB does not obtain data from all nodes, but pushes down the related calculation to other nodes. The execution plan is as follows:
+
+{{< copyable "sql" >}}
+
+```sql
+desc select count(*) from information_schema.cluster_slow_query where user = 'u1';
+```
+
+```
++--------------------------+----------+-----------+--------------------------+------------------------------------------------------+
+| id                       | estRows  | task      | access object            | operator info                                        |
++--------------------------+----------+-----------+--------------------------+------------------------------------------------------+
+| StreamAgg_20             | 1.00     | root      |                          | funcs:count(Column#53)->Column#51                    |
+| └─TableReader_21         | 1.00     | root      |                          | data:StreamAgg_9                                     |
+|   └─StreamAgg_9          | 1.00     | cop[tidb] |                          | funcs:count(1)->Column#53                            |
+|     └─Selection_19       | 10.00    | cop[tidb] |                          | eq(information_schema.cluster_slow_query.user, "u1") |
+|       └─TableFullScan_18 | 10000.00 | cop[tidb] | table:CLUSTER_SLOW_QUERY | keep order:false, stats:pseudo                       |
++--------------------------+----------+-----------+--------------------------+------------------------------------------------------+
+```
+
+In the above execution plan, the `user = u1` condition is pushed down to other (`cop`) TiDB nodes, and the aggregate operator is also pushed down (the `StreamAgg` operator in the graph).
+
+Currently, because information of the system tables is not collected in the statistics, sometimes some aggregation operators cannot be pushed down, which results in slow execution. You can manually specify the SQL HINT for aggregation pushdown to push down the aggregation operators. For example:
```

Comment:
```suggestion
Currently, because statistics of the system tables are not collected, sometimes some aggregation operators cannot be pushed down, which results in slow execution. In this case, you can manually specify the SQL HINT to push down the aggregation operators. For example:
```